pkgs/data/fonts/klingon-piqad-hasta/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchzip }:
+
+let
+  pname = "klingon-piqad-hasta";
+  version = "203";
+in (fetchzip {
+  name = "${pname}-${version}";
+
+  url = "https://www.evertype.com/fonts/tlh/${pname}.zip";
+
+  sha256 = "1f6l2dpha4kpm2fdw6b5lpzsjpzgvzw8sjh1qygir68rin99h4ac";
+
+  meta = with lib; {
+    description =
+      "Klingon pIqaD HaSta ‘viewscreen writing’ is a standard Klingon typeface";
```

do not duplicate pname in description:
```suggestion
    description =
      "A standard Klingon typeface";
```